api/v1alpha1/appgroup_types.go

```diff
@@ -17,6 +17,12 @@ import (
 const (
 	DefaultProgressingRequeue = 5 * time.Second
 	DefaultSucceededRequeue   = 5 * time.Minute
+
+	AppGroupNameKey   = "appgroup"
+	AppGroupFinalizer = "application-group-finalizer"
+
+	LastSuccessfulAnnotation = "orkestra/last-successful-applicationgroup"
+	ParentChartAnnotation    = "orkestra/parent-chart"
```

Comment:
Can we make these lower case since this package does not get imported elsewhere in the code

Reply:
We can't do this because the annotations are accessed outside of this package